evalml/automl/automl_search.py

```diff
@@ -593,7 +613,7 @@ def _add_baseline_pipelines(self, X, y):
             baseline = ModeBaselineBinaryPipeline(parameters={})
         elif self.problem_type == ProblemTypes.MULTICLASS:
             baseline = ModeBaselineMulticlassPipeline(parameters={})
-        elif self.problem_type == ProblemTypes.REGRESSION:
+        else:
```

Comment:
Ah, got it. This is great. I do wonder if we'll want to update our timeseries "baseline" to a weighted moving average or something. We can wait and see!

Reply:
Yep, lots of options here! Another naive thing we could do is just use the previous target value for "today's" prediction.